eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -418,8 +418,8 @@
         },
         {
           "regex": true,
-          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
-          "old": ".*? com\\.azure\\.resourcemanager\\.(cosmos|network|mobilenetwork)\\.models.*",
+          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded|removed)",
+          "old": ".*? com\\.azure\\.resourcemanager\\.(cosmos|network|mobilenetwork|imagebuilder)\\.models.*",
```

Comment:
Could you change this pattern to all resourcemanager models `".*? com\\.azure\\.resourcemanager\\.*\\.models.*"` (here is example, please check the regex)?

Also change the reason to `"Migration to azure-json"`

All future mgmt SDK release would need this as migration to azure-json

Reply:
Fixed in the new version.